### PR TITLE
Initialize project

### PR DESCRIPTION
**Pull request by Sahar-AbdelSamad**
IN this PR I:
- Initialised `React app`
- Installed `React Redux`
- Structured your application files using a "feature folder" approach
- Used the `ducks pattern` for your Redux files
- Added `React Router` and set two `<Route>s`: `Books` - the default view and `Categories`
- Added minimum styling
- Update README file

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    </ul>
+    <hr />
+    <h2 className="add-book">Add new Book</h2>
+    <form className="form">
+      <input type="text" placeholder="Book title" />
+      <input type="select" placeholder="Category" />
+      <input type="submit" className="submit" value="Add Book" />
+    </form>
+  </div>
+);
```
Comment (mikethreels) [inline src/components/Books.js]: - [x] In this component right now you are just displaying HTML code it doesn't really function as intended. the books component should render the `Book` and `CreateBook` components.
- the `Book` component takes the book data as a prop and renders the title and the author and has a button for removing books
- the `CreateBook` component renders a form with inputs for the title and the author of the book and has a button to submit
Both the button for removing books and for submitting the form should be placeholder buttons right now they don't have to do anything.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Sahar, 

Good job setting up the routes for this project, and very nicely done putting the navbar in a separate component 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 1 above)
Comment (Sahar-AbdelSamad) [reply src/components/Books.js]: Thank you :+1:  I implemented the necessary changes

**Review by er-contreras — APPROVED** [review]
Hi @Sahar-AbdelSamad , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- I see that you did most of the work, well done!

Felix

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**